set.mm

```diff
@@ -648705,6 +648801,29 @@ number axioms (add ~ ax-10 , ~ ax-11 , ~ ax-13 , ~ ax-nul , and remove
       BFABABDQZAHBARUJESUBZAUHFGUITGUIFGCAUIBTUKDUCUHUDUEUAEBUFUG $.
   $}
 
+  ${
+    cxpgt0d.1 $e |- ( ph -> A e. RR+ ) $.
+    cxpgt0d.2 $e |- ( ph -> N e. RR ) $.
+    $( Exponentiation with a positive mantissa is positive.  (Contributed by
```

Comment:
```suggestion
    $( Exponentiation with a positive base is positive.  (Contributed by
```
more standard and less ambiguous.
It could be more simply
> A positive real raised to a real power is positive.